static/src/javascripts/projects/common/modules/experiments/article-containers-test.js

```diff
@@ -0,0 +1,57 @@
+define([
+    'fastdom',
+    'common/utils/$',
+    'common/utils/_',
+    'common/utils/ajax',
+    'common/utils/mediator',
+    'common/utils/config',
+    'common/modules/experiments/ab'
+], function (
+    fastDom,
+    $,
+    _,
+    ajax,
+    mediator,
+    config,
+    ab
+) {
+    var targetEl = '.js-related';
+
+    function loadContainer(test) {
+        ajax({
+            url: test.endpoint,
+            type: 'json',
+            crossOrigin: true
+        })
+        .then(function (res) {
+            var el;
+
+            if (res.html) {
+                el = $.create(res.html);
```

Comment:
No - because the element is not getting attached to the DOM here. But it is a few lines later, where it _is_ fastDom'd.